docs/website/Makefile

```diff
@@ -1,21 +1,22 @@
 TOP=../../..
-WEBSITE_DOCS ?= $(TOP)/documentation
+WEBSITE_DOCS ?= $(TOP)/xamarin-docs-pr
 XAMARIN_ANALYSIS ?=$(TOP)/xamarin-analysis/shared/Xamarin.Analysis
 XAMARIN_ANALYSIS_DOC = xamarin-analysis-doc-tool/bin/Debug
 
 all: xamarin-analysis-doc
-	cp mtouch-errors.md $(WEBSITE_DOCS)/guides/ios/troubleshooting/mtouch-errors/index.md
+	cp mtouch-errors.md $(WEBSITE_DOCS)/docs/ios/troubleshooting/mtouch-errors.md
 	cp binding_objc_libs.md $(WEBSITE_DOCS)/guides/cross-platform/macios/binding/objective-c-libraries/index.md
-	cp binding_types_reference_guide.md $(WEBSITE_DOCS)/guides/cross-platform/macios/binding/binding-types-reference/index.md
-	cp xamarin-ios-analysis.md $(WEBSITE_DOCS)/guides/ios/troubleshooting/xamarin-ios-analysis/index.md
+	cp binding_types_reference_guide.md $(WEBSITE_DOCS)/docs/cross-platform/macios/binding/binding-types-reference.md
+	cp xamarin-ios-analysis.md $(WEBSITE_DOCS)/ios/troubleshooting/xamarin-ios-analysis.md
 	cp optimizations.md $(WEBSITE_DOCS)/guides/cross-platform/macios/build-optimizations/index.md
```

Comment:
Staged the build optimizations content in a PR here: https://github.com/MicrosoftDocs/xamarin-docs-pr/pull/204.

Build optimizations will live here:

docs/cross-platform/macios/optimizations.md